packages/vue-app/package.json

```diff
@@ -3,13 +3,13 @@
   "version": "2.3.2",
   "repository": "nuxt/nuxt.js",
   "license": "MIT",
-  "typings": "types/index.d.ts",
   "files": [
     "dist",
     "template",
     "types/*.d.ts"
   ],
   "main": "dist/vue-app.js",
+  "typings": "types/index.d.ts",
```

Comment:
Is this change necessary?

Reply:
@manniL not sure about this. Its done automatically when i do just do build, so i guess yes, unless its done inverse for someone other :)

Reply:
My bad when merging original typings PR. Our home-made pkg script sorts fields inside `package.json` so I confirm it is valid.